.eslintrc.js

```diff
@@ -6,7 +6,19 @@ module.exports = {
     'react/no-string-refs': 1,
     // Avoid errors about `UNSAFE` lifecycles (e.g. `UNSAFE_componentWillMount`)
     'react/no-deprecated': 0,
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      { argsIgnorePattern: '^_', ignoreRestSiblings: true },
+    ],
   },
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: {
+        'valid-jsdoc': 0,
```

Comment:
Valid JSDoc is redundant with TypeScript signatures. I disabled this rule for TS files because it's still convenient to use JSDoc-style comments to explain a function.

Example:

```js
/**
 * Blabla...
 */
function (...args) {}
```